neonvm/tools/vm-builder/files/Dockerfile.img

```diff
@@ -61,19 +61,47 @@ RUN set -e \
     && wget https://packages.timber.io/vector/0.26.0/vector-0.26.0-x86_64-unknown-linux-musl.tar.gz -O - \
     | tar xzvf - --strip-components 3 -C /neonvm/bin/ ./vector-x86_64-unknown-linux-musl/bin/vector
 
+# chrony
+RUN set -e \
+       && apk add --no-cache --no-progress --quiet \
+               chrony \
+       && mv /usr/sbin/chronyd /neonvm/bin/ \
+       && mv /usr/bin/chronyc  /neonvm/bin/ \
+       && cp -f /lib/libc.musl-x86_64.so.1 /neonvm/lib/ \
+       && cp -f /lib/libz.so.1 /neonvm/lib/ \
+       && cp -f /usr/lib/libcap.so.2 /neonvm/lib/ \
+       && cp -f /usr/lib/libffi.so.8 /neonvm/lib/ \
+       && cp -f /usr/lib/libgmp.so.10 /neonvm/lib/ \
+       && cp -f /usr/lib/libgnutls.so.30 /neonvm/lib/ \
+       && cp -f /usr/lib/libhogweed.so.6 /neonvm/lib/ \
+       && cp -f /usr/lib/libnettle.so.8 /neonvm/lib/ \
+       && cp -f /usr/lib/libp11-kit.so.0 /neonvm/lib/ \
+       && cp -f /usr/lib/libtasn1.so.6 /neonvm/lib/ \
+       && cp -f /usr/lib/libunistring.so.2 /neonvm/lib/
+
 # init scripts
 COPY inittab     /neonvm/bin/inittab
 COPY vminit      /neonvm/bin/vminit
 COPY vmstart     /neonvm/bin/vmstart
 COPY vmshutdown  /neonvm/bin/vmshutdown
 COPY vmacpi      /neonvm/acpi/vmacpi
 COPY vector.yaml /neonvm/config/vector.yaml
+COPY chrony.conf /neonvm/config/chrony.conf
 RUN chmod +rx /neonvm/bin/vminit /neonvm/bin/vmstart /neonvm/bin/vmshutdown
 
+# rootdisk modification
+FROM rootdisk AS rootdisk-mod
```

Comment:
this `rootdisk-mod` is the same change as in #726, right?
out of curiosity, what's the plan w.r.t. these two PRs?

Reply:
I'll resolve the conflict manually.